### PR TITLE
Adding delete file page to webapp. Adding link to other pages.

### PR DESCRIPTION
**Pull request by frostyclawz**
Delete configuration page is being added to the overall webapp. Adding links to the page on the previous three pages in the webapp.

### CODE REVIEW

**Comment by frostyclawz** [comment]
Screenshot 2020-07-06 at 4 29 03 PM - Display 2

File: src/main/webapp/deletefile.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+#links {
+  text-align: center;
+  border: solid 3px black;
+  margin: auto;
+  width: 500px;
+}
+
+#deleteform{
```
Comment (cadecrandall) [inline src/main/webapp/deletefile.css:13]: nit: can you add a space before the opening brace?

File: src/main/webapp/deletefile.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="deletefile.css">
```
Comment (cadecrandall) [inline src/main/webapp/deletefile.html:6]: can you rename your IDs to be delimited by a hyphen?

File: src/main/webapp/deletefile.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <a href="createfile.html">Create Configuration</a>
+    </div><br>
+    <div id="deleteform">
+      <form>
+        <span>Customer ID:</span>
+        <input type="text"><br>
+        <span>Account ID:</span>
+        <input type="text"><br>
+        <input type="submit">
+      </form>
```
Comment (cadecrandall) [inline src/main/webapp/deletefile.html:23]: should these input elements have IDs associated with them?

File: src/main/webapp/deletefile.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="deletefile.css">
+    <!-- Bootsrap CSS -->
```
Comment (ianjjohnson) [inline src/main/webapp/deletefile.html:7]: "bootstrap" (spelling)

File: src/main/webapp/deletefile.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" 
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <script src="script.js"></script>
```
Comment (ianjjohnson) [inline src/main/webapp/deletefile.html:10]: Please rename "script.js" to something more meaningful. I don't see that script anywhere, so maybe this is just accidental copypasta?

File: src/main/webapp/deletefile.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  <body>
+    <h1>Billing Configuration Sync Tool</h1>
+    <h2>Delete Configuration</h2>
+    <div id="links">
+      <a href="readfile.html">Read Configuration</a>
+      <a href="editfile.html">Edit Configuration</a>
+      <a href="createfile.html">Create Configuration</a>
+    </div><br>
+    <div id="delete-search">
+      <form onsubmit="openModal()" id="deleteForm">
```
Comment (ianjjohnson) [inline src/main/webapp/deletefile.html:28]: this goes with one of Cade's comments: consider 'delete-form'

File: src/main/webapp/deletefile.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+        <span>Customer ID:</span>
+        <input type="text"><br>
+        <span>Account ID:</span>
+        <input type="text"><br>
+        <button type="button" class="btn-primary" data-toggle="modal"
+        data-target="#deleteModal">Submit</button>
+      </form>
+    </div>
+    <!-- TODO: Add modal confirmation. -->
+    <div class="modal fade" tabindex="-1" role="dialog" id="deleteModal">
```
Comment (ianjjohnson) [inline src/main/webapp/deletefile.html:38]: "modal fade" -> "modal-fade"?

File: src/main/webapp/deletefile.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+            <button type="button" class="close" data-dismiss="modal"
+              aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <h4 class="modal-title">Delete Configuration</h4>
+          </div>
+          <div class="modal-body">
+            <p>Are you sure you want to delete this configuration?</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn-default" data-dismiss="modal">Cancel</button>
+            <button type="button" class="btn-primary">Okay</button>
```
Comment (ianjjohnson) [inline src/main/webapp/deletefile.html:51]: Okay -> Yes?

File: src/main/webapp/deletefile.html (hunk 5 above)
Comment (jakecarlson1) [reply src/main/webapp/deletefile.html:10]: +1, please rename to deletescript.js (or delete-script.js if you are introducing hyphenated file names).

File: src/main/webapp/deletefile.html (hunk 7 above)
Comment (jakecarlson1) [reply src/main/webapp/deletefile.html:37]: +1, modal fade to modal-fade will make this consistent with the class fields below.

Comment (jakecarlson1) [reply src/main/webapp/deletefile.html:39]: Thank you for the comment clarifying this.

**Pull request merged**